platform/api/packet/api.go

```diff
@@ -63,14 +63,16 @@ var (
 		"arm64-usr": "https://alpha.release.core-os.net/arm64-usr/current/coreos_production_packet_image.bin.bz2",
 	}
 	defaultPlan = map[string]string{
-		"amd64-usr": "baremetal_0",
-		"arm64-usr": "baremetal_2a",
-		"s390x-usr": "baremetal_3a",
+		"amd64-usr":   "baremetal_0",
+		"arm64-usr":   "baremetal_2a",
+		"s390x-usr":   "baremetal_3a",
+		"ppc64le-usr": "baremetal_4a",
```

Comment:
Is this an actual packet slug?

Reply:
could you elaborate?

Reply:
Is `baremetal_4a` actually a Packet machine type for a ppc64le machine?

Reply:
Honestly I have no idea.

Reply:
This can be dropped as well then. To my knowledge packet doesn't have a ppc64le machine type.

Reply:
Ack, thanks.